test/e2e/operator_helper_test.go

```diff
@@ -39,7 +39,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/kind/pkg/apis/config/v1alpha4"
-	"sigs.k8s.io/kind/pkg/cluster"
+	kind_cluster "sigs.k8s.io/kind/pkg/cluster"
```

Comment:
why not simply?

```suggestion
	kind "sigs.k8s.io/kind/pkg/cluster"
```

I have never seen using `_` in aliases, I suspect it is not idiomatic.

Reply:
the more idiomatic thing would be `kindcluster`, changing.

Reply:
done, PTAL